events/code_review_events/workflow.py

```diff
@@ -32,21 +32,26 @@
 from code_review_events import taskcluster_config
 from code_review_events.bugbug_utils import BugbugUtils
 from code_review_tools import heroku
+from code_review_tools.lando import LandoWarnings
 
 logger = structlog.get_logger(__name__)
 
 PULSE_TASK_GROUP_RESOLVED = "exchange/taskcluster-queue/v1/task-group-resolved"
 PULSE_TASK_COMPLETED = "exchange/taskcluster-queue/v1/task-completed"
 PULSE_TASK_FAILED = "exchange/taskcluster-queue/v1/task-failed"
 
+LANDO_WARNING_MESSAGE = (
+    "Review time linters are still running. Are you sure you want to continue?"
```

Comment:
`Static analysis and linting is still in progress` (no need for the second part "Are you sure you want to continue?" as this part is already in the Lando UI itself)